### PR TITLE
 Parsing Matrices to LaTeX. Tries to fix issue #191

### PR DESCRIPTION
**Pull request by VaishnaviGanamukkala**
Went through the contributing guideline, added tests, and ran all the tests. Please review and let me know if any changes are required.

Thanks.

### CODE REVIEW

File: visma/io/parser.py [hunk 1]
```diff
@@ -52,7 +52,20 @@ def tokensToLatex(eqTokens):
     """
     eqLatex = ""
     for token in eqTokens:
-        eqLatex += token.__str__()
+        if isinstance(token, Matrix):
+            eqLatex += "\\begin{bmatrix}"
```
Comment (8hantanu) [inline visma/io/parser.py:56]: ~I think there will be a single backslash.~
**Edit**: My bad. I forgot the extra backslash is used for escaping. 
```suggestion
            eqLatex += "\begin{bmatrix}"
```

File: visma/io/parser.py [hunk 2]
```diff
@@ -52,7 +52,20 @@ def tokensToLatex(eqTokens):
...
+            for row in token.value:
+                for column in row:
+                    for term in column:
+                        eqLatex += term.__str__()
+                    if row.index(column) < len(row) - 1:
+                        eqLatex += '&'
+                    elif row.index(column) == len(row) - 1 and token.value.index(row) < len(token.value) - 1:
+                        eqLatex += '\\\\'
+            eqLatex += "\\end{bmatrix}"
```
Comment (8hantanu) [inline visma/io/parser.py:65]:
```suggestion
            eqLatex += "\end{bmatrix}"
```

File: visma/io/parser.py [hunk 3]
```diff
@@ -52,7 +52,20 @@ def tokensToLatex(eqTokens):
...
+        if isinstance(token, Matrix):
+            eqLatex += "\\begin{bmatrix}"
+            for row in token.value:
+                for column in row:
+                    for term in column:
+                        eqLatex += term.__str__()
+                    if row.index(column) < len(row) - 1:
+                        eqLatex += '&'
+                    elif row.index(column) == len(row) - 1 and token.value.index(row) < len(token.value) - 1:
+                        eqLatex += '\\\\'
```
Comment (8hantanu) [inline visma/io/parser.py:64]:
```suggestion
                        eqLatex += '\\'
```

**Review by 8hantanu — COMMENTED** [review]
You can verify the LaTeX output [here](https://www.mathjax.org/#demo).

**Review by 8hantanu — APPROVED** [review]

**Comment by mayankDhiman** [comment]
Thanks for the contribution.

**Pull request merged**